phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/installations/InstallationsController.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2015-2017 PÂRIS Quentin
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+package org.phoenicis.javafx.controller.installations;
+
+import org.phoenicis.javafx.views.mainwindow.installations.ViewInstallations;
+
+public class InstallationsController {
```

Comment:
I wanted to have so that the same structure is used everywhere. Maybe we decide to have a module `phoenicis-installations` with the dtos and an `InstallationsManager` or something like that. Then it will be good to have the controller.